ui/src/app/pages/login/account-creation/account-creation-tos.spec.tsx

```diff
@@ -0,0 +1,91 @@
+import {mount, ReactWrapper, shallow, ShallowWrapper} from 'enzyme';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+import AccountCreationTos, {AccountCreationTosProps} from './account-creation-tos';
+import {Document, Page, pdfjs} from 'react-pdf';
+
+type AnyWrapper = (ShallowWrapper|ReactWrapper);
+const getPrivacyCheckbox = (wrapper: AnyWrapper): AnyWrapper => {
+  return wrapper.find('[data-test-id="privacy-statement-check"]');
+};
+const getTosCheckbox = (wrapper: AnyWrapper): AnyWrapper => {
+  return wrapper.find('[data-test-id="terms-of-service-check"]');
+};
+const getNextButton = (wrapper: AnyWrapper ): AnyWrapper => {
+  return wrapper.find('[data-test-id="next-button"]');
+};
+
+let props: AccountCreationTosProps;
+const onCompleteSpy = jest.fn();
+
+beforeEach(() => {
+  jest.clearAllMocks();
+  props = {
+    windowSize: {width: 1700, height: 0},
+    onComplete: onCompleteSpy,
+    pdfPath: '/assets/documents/terms-of-service.pdf'
```

Comment:
Presumably this last part wasn't meant to be included in the TOS content itself, as of now it's super confusing since we'd show multiple sets of checkboxes, the ones in the PDF of course are not interactive: 
![image](https://user-images.githubusercontent.com/822298/73900232-f776fb00-4843-11ea-9909-2aca222c460e.png)